src/Evolution/Systems/NewtonianEuler/Subcell/PrimitiveGhostData.hpp

```diff
@@ -42,4 +49,48 @@ class PrimitiveGhostDataOnSubcells {
   static Variables<prims_to_reconstruct_tags> apply(
       const Variables<prim_tags>& prims) noexcept;
 };
+
+/*!
+ * \brief Projects the mass density, velocity, and pressure to the subcells so
+ * they can be sent to the neighbors for their reconstructions.
+ *
+ * The data on the subcells is sliced before sending to the neighbors. That is,
```

Comment:
I was confused by this as it does not appear to me that the slicing is done in this function